client/src/components/byline-profiles/index.js

```diff
@@ -99,12 +99,10 @@ const BylineProfiles = ({
   const removeItem = (id) => {
     const index = profiles.findIndex((item) => item.id === id);
     if (0 <= index) {
-      this.setState({
```

Comment:
I honestly don't know how this was ever working...probably it wasn't

Reply:
I noticed when bisecting that it was recently converted from a `React.Component` class to a function, so probably only broken as of recently

Reply:
yeah, figured that must be the case